hrui/templates/CRM/common/footer.tpl

```diff
@@ -28,17 +28,29 @@
   {if !empty($contactId)}
     {include file="CRM/common/contactFooter.tpl"}
   {/if}
-
   <div class="crm-footer" id="civicrm-footer">
     {* PCHR-1323 - Display CiviHR version info. *}
     {ts}Powered by CiviHR {/ts} {civihrVersion}.
 
+    {php}
+      if(user_access('view system status on footer')):
+    {/php}
+
     {if !empty($footer_status_severity)}
-      <span class="status{if $footer_status_severity gt 3} crm-error{elseif $footer_status_severity gt 2} crm-warning{else} crm-ok{/if}">
+      <span
```

Comment:
some of the alignment looks strange here, I know they're long lines, but maybe it makes sense to keep the text on the same line and only separate by dom elements, e.g.

```
<div>
  <span class"foo">long line of text long line of text long line of text long line of text long line of text long line of text long line of text long line of text </span>
</div>